openlibrary/i18n/es/messages.po

```diff
@@ -128,8 +128,8 @@ msgid "Passwords didn't match."
 msgstr "Las contraseñas no coinciden."
 
 #: forms.py:62
-msgid "Send me a monthly newsletter from the <a href=\"https://archive.org/\">Internet Archive</a>, the non-profit that runs Open Library"
-msgstr "Envíame un boletín mensual de <a href=\"https://archive.org/\">Internet Archive</a>, la organización sin fines de lucro que administra Open Library"
+msgid "I want to receive news, announcements, and resources from the <a href=\"https://archive.org/\">Internet Archive</a>, the non-profit that runs Open Library."
+msgstr "Envíame un boletín mensual de <a href=\"https://archive.org/\">Internet Archive</a>, la organización sin fines de lucro que administra Open Library."
```

Comment:
We might need to update the translation as well.

Reply:
👍 @dcapillae Any chance you could update this? If you leave a suggestion on GitHub, I can include it as part of this PR :)

Reply:
```suggestion
msgstr "Quiero recibir noticias, anuncios y recursos de <a href=\"https://archive.org/\">Internet Archive</a>, la organización sin fines de lucro que administra Open Library."
```

Reply:
Muchas gracias @dcapillae !